lib/Controller/LockController.php

```diff
@@ -94,7 +96,9 @@ public function locking(string $fileId): DataResponse {
 			$user = $this->userSession->getUser();
 			$file = $this->fileService->getFileFromId($user->getUID(), (int)$fileId);
 
-			$lock = $this->lockService->lockFile($file, $user);
+			$lock = $this->lockService->lock(new LockContext(
```

Comment:
Any reason to name this controller's methods "locking" and "unlocking" instead of "lock" and "unlock"?